src/private-key.ts

```diff
@@ -21,6 +23,9 @@ function findPrivateKey (filepath: string): Buffer | string {
   if (filepath) {
     return fs.readFileSync(filepath)
   }
+  if (process.env.PRIVATE_KEY_BASE64) {
+    return Buffer.from(process.env.PRIVATE_KEY_BASE64, 'base64').toString()
```

Comment:
What is the error if this is not a valid base64 string?

Reply:
It still goes through the OpenSSL verification when we do `jwt.sign()`, so if it doesn't convert to a valid x.509 formatted certificate, you'll get the same errors you would if you had a malformed .pem file (e.g. `PEM routines:PEM_read_bio:no start line`). I tested a few other ways to "break" the key, like adding spaces, extra characters, deleting characters, and adding `{}` or `()`. Depending on where your invalid characters are, it will either break the PEM format and not work, or those characters get ignored by `Buffer.from` based on [the spec](https://tools.ietf.org/html/rfc4648#section-5). I'm not sure if `is-base64` would catch anything more than that, since node and OpenSSL will already throw errors if something isn't right.

One piece of friction that could cause issues is we aren't providing a recommended way to convert the key from the GitHub App to base64. One idea I had there is we could provide an `encode-key` script that takes the .pem file and throws the encoded version in your `.env` file.